psiturk/example/static/css/style.css

```diff
@@ -11,67 +11,80 @@ body {
 
 /* text styling */
 strong {
-  font-weight: 600;         
+    font-weight: 600;         
 }
 
 .warm {
-  color: steelblue;
-  text-align: justify;
+    color: steelblue;
+    text-align: justify;
 }
 
 .cool {
-  color: #ccccff;
+    color: #ccccff;
 }
 
 h1 {
-  margin-top: 0px;
+    margin-top: 0px;
 }
 
-
 /* these are all the same for now, but easy to modify individually */
 #container-ad, #container-error, #container-exp, #container-consent, #container-instructions, #container-questionnaire {
+    position: absolute;
```

Comment:
Oh I get it now. the `style.css` on psiturk.org, which I incorporated into this commit, is outdated. I'll just revert all changes to this file.